support-frontend/README.md

```diff
@@ -1,8 +1,46 @@
+# Support Frontend
 Frontend for the new [supporter platform](https://support.theguardian.com/).
 
-### SSH
+## How tos
+
+### How to SSM into EC2 servers
 Use SSM Tunnel - [ssm-scala](https://github.com/guardian/ssm-scala#enabling-ssm-tunnel):
 
-```
+```Bash
 ssm ssh --profile membership --tags frontend,support,CODE -a -x --newest --ssm-tunnel
 ```
+
+---
+
+### How to: Using campaign switches
+We often run campaigns that have certain features or modifications enabled across the site for the length of that
+campaign.
+
+To add or remove these features without code changes, allowing for quick and out of hours releasing, we use the Campaign
+switches on the Reader [Revenue Support Console (RRCP) switchboard](https://support.gutools.co.uk/switches).
+
+This allows for sending these changes to colleagues for testing using a URL hash. E.g.
+https://support.theguardian.com/us/contribute#settings.switches.campaignSwitches.usEoy2023=On
+
+The switch is only enabled for that browser session and is not persisted.
+
+To create a new campaign
+1. Go to the [RRCP switchboard](https://support.gutools.co.uk/switches)
+1. Create a new switch under "Campaign switches"
+1. Save the switchboard
+1. Add the switch to [Switches.scala](./app/admin/settings/Switches.scala) under `CampaignSwitches` as
+`switchName: Option[Switch] = None`
+
+![An screenshow of the steps to adding a campaign switch in RRCP](./wiki-images/add-campaign-switches.png)
+
+#### CODE / Local testing
+To test in code, you would need to follow step 1. on
+[the CODE switchboard](https://support.code.dev-gutools.co.uk/switches).
+
+To test locally, copy the `switches_v2.json` file stored in S3 to the specified path in
+[`DEV.public.conf`](./conf/DEV.public.conf) ie. `~/.gu/support-admin-console/switches_v2.json`.
+
+```bash
+# Make sure you have Janus credentials and `~/.gu/support-admin-console` exists
+aws s3 cp s3://$SUPPORT_ADMIN_BUCKET/ENV/switches_v2.json  ~/.gu/support-admin-console/switches_v2.json --profile membership
+```
```

Comment:
I am more than happy to export this to the Wiki if people think it should live there instead.

Reply:
Hey, this is great, thanks for doing it ... I'd like to add my vote for putting it in the wiki ... although maybe at some point a contents list of links to the most used/helpful stuff in the wiki might be nice in here.

Reply:
Its definitely worth having a think about how we manage our knowledge. The main reason this felt useful here is it's a code thing, and so when I am in my IDE, I can just reference the markdown. I can also 

The wiki itself is also very out of date in some instances, making it hard to know what information to follow and what not to, which I have seen often happen when it is decoupled from the code where it's relevant. 

The other piece is I would, as the author, really appreciate if these instructions are easy to follow, and a PR is a great place for this, so I might leave the content in here until people have reviewed and validated it's good. I am not sure what a good process for this is in the Wiki?

I can then remove it and push it to the wiki.

Reply:
I've added it here for now:
https://github.com/guardian/support-frontend/wiki/Quickstart:-Campaign-feature-development

Reply:
I've removed this from here for now. I think we should whack some Wiki links into the main `README`s as they do seem like the main source of information, but are not massively discoverable from the codebase.